docs/chapters/baseplatform/creating-sdk.rst

```diff
@@ -0,0 +1,14 @@
+Building the SDK installer
+===========================
+
+Building the SDK installer is not very different from creating the corresponding
+target image. All you have to do, is pass an additional option to bitbake: 
+
+.. code-block:: bash
+
+    bitbake -c populate_sdk <image>
+
+This command will create a toolchain installer that contains the sysroot that
+matches your target root filesystem. The installer will be under tmp/deploy/sdk
```

Comment:
Hmm.. not sure. When you bitbake, you are definitely in the build dir so I don't see any chance of user getting confused on where it is.